<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Engineering Diffraction Analysis - Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid grey;border-radius: 10px;margin-top: 1ex; margin-right: 0ex
}
QGroupBox:title {
                          subcontrol-origin: margin;
                          subcontrol-position: top center;
                          padding-top: 0px;
                          padding-bottom: 0px;
							  padding-left: 5px;
                          padding-right: 5px;
                          color: rgb(56, 56, 56)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_general">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="FileFinderWidget" name="finder_save" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_calib">
     <property name="title">
      <string>Calibration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="check_vanRecalc">
        <property name="text">
         <string>Force Vanadium Recalculation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="FileFinderWidget" name="finder_fullCalib" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fitting">
     <property name="title">
      <string>Fitting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
	  <item row="0" column="0">
	   <widget class="QLabel" name="log_list_label"></widget>
	  </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="log_list">
		<property name="verticalScrollBarPolicy">
		  <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
	  <item row="2" column="0">
	   <widget class="QLabel" name="primary_log_label"></widget>
	  </item>
	  <item row="3" column="0">
       <widget class="QComboBox" name="primary_log">
       </widget>
      </item>
	  <item row="3" column="1">
	   <layout class="QVBoxLayout" name="SeqFitOrder_layout">
	    <item>
         <widget class="QCheckBox" name="check_ascending">
          <property name="text">
           <string>Ascending</string>
          </property>
		  <property name="checked">
           <bool>true</bool>
          </property>
	    </widget>
	    </item>
		<item>
         <widget class="QCheckBox" name="check_descending">
          <property name="text">
           <string>Descending</string>
          </property>
		  <property name="checked">
           <bool>false</bool>
          </property>
	    </widget>
	    </item>
	   </layout>
	  </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_apply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>mantidqt.widgets.filefinderwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
